src/tests/class_tests/openms/source/SvmTheoreticalSpectrumGenerator_test.cpp

```diff
@@ -95,7 +95,13 @@ START_SECTION(void simulate(PeakSpectrum &spectrum, const AASequence &peptide, b
   PeakMap exp;
   MzMLFile mz_file;
 
+#if OPENMS_BOOST_VERSION_MINOR < 56
   mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test.mzML"),exp);
+  TEST_EQUAL(spec.size(), 7);
+#else
+  mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test_boost58.mzML"),exp);
+  TEST_EQUAL(spec.size(), 8);
+#endif
```

Comment:
we are simulating the peptide `IFSQVGK` which should have 7 fragment ions, right? We now get an additional ion at 
216.134 which is a y4++ ion that does not appear with the previous boost...